src/lib/Scenes/Artwork/Components/CommercialButtons/CommercialButtons.tsx

```diff
@@ -35,6 +35,8 @@ export class CommercialButtons extends React.Component<CommercialButtonProps> {
     const { isBuyNowable, isAcquireable, isOfferable, isInquireable, isInAuction, editionSets, isForSale } = artwork
     const noEditions = (editionSets && editionSets.length === 0) || !editionSets
 
+    const artworkSaleEnded = artwork.sale && artwork.sale.isClosed
+
     if (isInAuction && artwork.sale && auctionState !== "hasEnded" && isForSale) {
```

Comment:
Question: is there a difference between this `auctionState !== "hasEnded"` and `artwork.sale.isClosed`? If not isn't it better to update that to also use `isClosed`?

Reply:
We prefer to use `auctionState` for this because it will change client-side depending on the status of the timer.

When `auctionState` is instantiated, it will take into account whether the auction `isClosed`.